internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -63,6 +65,10 @@ func (handler StorageHandler) Start() error {
 		return fmt.Errorf("unable to add required route: %s: %s", apiResourceRoute, err.Error())
 	}
 
+	if err := handler.service.AddRoute(apiResourceRoute, handler.addContext(deviceHandler), http.MethodGet); err != nil {
```

Comment:
Please revise line 64 like the suggestion instead of adding new code.

```suggestion
	if err := handler.service.AddRoute(apiResourceRoute, handler.addContext(deviceHandler), http.MethodPost, http.MethodGet); err != nil {
```

Reply:
Thank you, Done.